### ISSUE TITLE
Mod Icon and 3 dots at the left showing on a draft article

### ISSUE THREAD

**Issue by reyesvicente** [forem/forem#19305]
**Describe the bug**
The mod icon and 3 dots at the left is showing on a draft post

**To Reproduce**
Create a new article and save it as draft. Next, edit the draft article and it will show you the mod icon and the 3 dots at the left

**Expected behavior**
No mod icon and 3 dots at the left of the draft article

**Screenshots**
![Screenshot from 2023-04-07 18-50-33](https://user-images.githubusercontent.com/45283313/230597690-b6a2be02-6b64-4a11-bffd-ab769a46bbe6.png)

**Desktop (please complete the following information):**

- OS, version: Linux Ubuntu 22.04
- Browser, version: [Version 1.50.114 Chromium: 112.0.5615.49 (Official Build) (64-bit)](https://brave.com/latest/)

**Comment by FrancisTRdev** [comment]
Will work on this @benhalpern

**Comment by FrancisTRdev** [comment]
@mikeydorje Notice you assigned yourself to this issue back In Jan, but haven't seen any updates. Hope that's okay if I can work on it!

**Comment by mikeydorje** [comment]
@FrancisTRAlt Yes, go ahead! I had assigned myself to take a look later but got onto other things.

**Comment by FrancisTRdev** [comment]
@mikeydorje Is there a way where I have the ability to assign myself to these issues? Sometimes I get loss when I want to work on multiple issues and I have to copy/paste the link to my personal notes. Is it possible to invite me to the repository?

**Comment by mikeydorje** [comment]
@FrancisTRAlt That's up to @benhalpern, but you could use a search in the issues field like `is:issue state:open commenter:@me`, or also, subscribe to the issue, so updates show in your inbox/notifications and then click on it from there, and there will be a "save" button at the top.

**Comment by FrancisTRAlt** [comment]
@mikeydorje Sounds good! Thanks for the help!
